src/public/modules/forms/admin/views/edit-fields.client.modal.html

```diff
@@ -905,7 +905,10 @@ <h2 class="modal-title">
             Options
           </div>
           <div class="col-xs-12 field-input dropdown-options">
-            <div ng-if="vm.field.fieldType === 'dropdown'" class="optionFrom">
+            <div
+              ng-if="vm.field.fieldType === 'dropdown' || vm.field.fieldType === 'checkbox'"
+              class="optionFrom"
+            >
               <textarea
                 class="input-custom input-medium"
                 rows="4"
```

Comment:
Can update the function name `vm.reloadDropdownField(vm.field)` to `vm.reloadFieldOptions(vm.field)`? Makes more sense now that its used by both dropdown and checkbox